core-api/src/main/java/com/sequenceiq/cloudbreak/api/model/ExecutorType.java

```diff
@@ -0,0 +1,5 @@
+package com.sequenceiq.cloudbreak.api.model;
+
+public enum ExecutorType {
+    CONTAINER, SIMPLE
```

Comment:
CONTAINER is the LinuxContainerExecutor, but SIMPLE sounds a bit strange, what is the official name for that in yarn? Can we name it accordingly?